src/kodeagent/fca.py

```
@@ -529,13 +530,41 @@ async def run(
-                **(self.litellm_params or {}),
-            )
+            try:
+                response = await litellm.acompletion(
+                    model=self.model_name,
+                    messages=self.chat_history,
+                    tools=self.tool_schemas,
+                    tool_choice='auto',
+                    **(self.litellm_params or {}),
+                )
+                consecutive_llm_errors = 0
+            except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)